README.md

```diff
@@ -0,0 +1,30 @@
+# CFPB GitHub Actions workflow templates
+
+This repo houses CFPB-specific GitHub Actions workflow templates.
+
+To use our workflow templates, click on the "Actions" tab in a CFPB GitHub repository. If the repository does not already have a GitHub Actions workflow, you will be presented with the option of creating one from one of our templates:
+
+![GitHub Actions template listing](github-actions-templates.png)
+
+If the repository already has a GitHub Actions workflow, click on the "New workflow" button at the top of the workflow list on the left. 
+
+![GitHub Actions New workflow button](github-actions-new-workflow.png)
+
+Then you will be presented with the option to create one from one of our templates, shown above.
+
+New workflow templates can be created in [github.com/cfpb/.github](https://github.com/cfpb/.github) based on [GitHub's workflow template documentation](https://docs.github.com/en/actions/configuring-and-managing-workflows/sharing-workflow-templates-within-your-organization).
+
+## consumerfinance.gov-specific workflow templates
+
+- [cfgov-frontend-ci.yml](workflow-templates/cfgov-frontend-ci.yml): A frontend CI workflow template for cf.gov satellite apps.
+- [cfgov-python-satellite-ci.yml](workflow-templates/cfgov-python-satellite-ci.yml): A backend CI workflow template for cf.gov satellite apps based on tox.
```

Comment:
What would you think about reframing these as general front-end/back-end CI workflows? They offer what we would consider to be best practices for any CFPB project.

Reply:
We can do that...it felt somewhat presumptuous to me to do that (at least on the backend because it presupposes Python).

Reply:
You can word it in such a way that it doesn't. Maybe instead of calling them frontend/backend, it should be JavaScript/Python?

Reply:
@Scotchester looking at this again, I'm fine moving `cfgov-frontend-ci` to `cfpb-javascript-ci`, but I think I'd like to keep the distinction between cfgov satellites and Python libraries, even if the only difference is the default of including Django in the tox matrix.